### PR TITLE
[ETP-423] Tweaks for IAC meta handling

### PR DESCRIPTION
**Pull request by sc0ttkclark**
[ETP-423]

Some minor tweaks for ET on IAC. One of which is some flags work so that Promoter has some more attendee flags to work with.

The other is a fix to the template caller for the orders screen so it can be properly hooked into by IAC.

Artifacts in https://github.com/moderntribe/event-tickets-plus/pull/1041

[ETP-423]: https://moderntribe.atlassian.net/browse/ETP-423

### CODE REVIEW

**Review by juanfra — DISMISSED** [review]

File: src/Tribe/REST/V1/Post_Repository.php [hunk 1]
```diff
@@ -962,6 +962,11 @@ protected function build_attendee_data( array $attendee, $context = 'default' )
 					'email'           => Tribe__Utils__Array::get( $attendee, 'holder_email', Tribe__Utils__Array::get( $attendee, 'purchaser_email', '' ) ),
 					'checked_in'      => $checked_in,
 					'checkin_details' => $checkin_details,
+
+					// Show Attendee flags.
+					// @todo Make these live in future IAC work.
+					'is_subscribed' => false,
```
Comment (tr1b0t) [inline src/Tribe/REST/V1/Post_Repository.php:968]: :warning: Array double arrow not aligned correctly; expected 3 space(s) between "'is_subscribed'" and double arrow, but found 1.

<sub>hash: fdc92cb4020865001f28fb8f9d3dd927</sub>

File: src/Tribe/REST/V1/Post_Repository.php [hunk 2]
```diff
@@ -962,6 +962,11 @@ protected function build_attendee_data( array $attendee, $context = 'default' )
...
+					'is_purchaser'  => true,
```
Comment (tr1b0t) [inline src/Tribe/REST/V1/Post_Repository.php:969]: :warning: Array double arrow not aligned correctly; expected 4 space(s) between "'is_purchaser'" and double arrow, but found 2.

<sub>hash: 769f38b9479112a21bafcf40f788c767</sub>

File: src/Tribe/RSVP.php [hunk 3]
```diff
@@ -2061,6 +2061,11 @@ public function get_attendee( $attendee, $post_id = 0 ) {
 
 			// Attendee Meta
 			'attendee_meta' => $meta,
+
+			// Handle initial Attendee flags.
+			// @todo Make these live in future IAC work.
+			'is_subscribed' => false,
```
Comment (tr1b0t) [inline src/Tribe/RSVP.php:2067]: :warning: Array double arrow not aligned correctly; expected 6 space(s) between "'is_subscribed'" and double arrow, but found 1.

<sub>hash: 78c1507629bc016e97ad198980952ddf</sub>

File: src/Tribe/RSVP.php [hunk 4]
```diff
@@ -2061,6 +2061,11 @@ public function get_attendee( $attendee, $post_id = 0 ) {
...
+			'is_purchaser'  => true,
```
Comment (tr1b0t) [inline src/Tribe/RSVP.php:2068]: :warning: Array double arrow not aligned correctly; expected 7 space(s) between "'is_purchaser'" and double arrow, but found 2.

<sub>hash: a941c82a89e26b48fcf792046ee1e879</sub>

File: src/views/tickets/orders.php [hunk 5]
```diff
@@ -33,6 +33,9 @@
 $user_id   = get_current_user_id();
 $provider  = Tribe__Tickets__Tickets::get_event_ticket_provider_object( $event_id );
 
+/** @var Tribe__Tickets__Editor__Template $template */
+$template  = tribe( 'tickets.editor.template' );
```
Comment (tr1b0t) [inline src/views/tickets/orders.php:37]: :warning: Equals sign not aligned correctly; expected 1 space but found 2 spaces

<sub>hash: a4c9be269d11396315b33c9f8f99a916</sub>

**Review by tr1b0t — COMMENTED** [review]
:warning: 5 warning(s)

File: src/views/tickets/orders.php (hunk 5 above)
Comment (sc0ttkclark) [reply src/views/tickets/orders.php:37]:
```suggestion
$template = tribe( 'tickets.editor.template' );
```

**Pull request merged**